default/python/README.md

```diff
@@ -24,27 +24,29 @@ planet types, planet sizes, and also other content get placed.
 
 # Code style check
 
-Code style checks are done with [pycodestyle](https://pypi.python.org/pypi/pycodestyle)
+We use [flake8-putty](https://pypi.python.org/pypi/flake8-putty)
+which include [flake8](https://pypi.python.org/pypi/flake8)
+which include [pycodestyle](https://pypi.python.org/pypi/pycodestyle) and other tools.
 
-This part is in progress.
+`flake8-putty` is `flake8` plugin that allows to setup specific rules on certain files, see `putty-ignore` section in `tox.ini`
```

Comment:
that allows the setup of specific rules...
that allows one to setup specific rules...
that allows specific rules to be setup...